packages/web-app-files/src/components/AppBar/ViewOptions.vue

```diff
@@ -130,6 +148,21 @@ export default defineComponent({
       }
     }
 
+    const setItemsPerPage = (itemsPerPage: string) => {
+      const resetPagination = itemsPerPage > unref(itemsPerPageQuery) && unref(currentPage) > 1
```

Comment:
IMO we should go back to page 1 whenever the items per page change. So not only if the items per page increase, but instead on any change

Reply:
I thought we already had this kind of reset in the past. Don't know if it got lost when we moved the pagination params from the store into the composable+query. 🤔 do you know or did you find any orphaned code fragments regarding the page reset?

Reply:
can't find anything either